app/src/components/submissionPage/SnapList.tsx

```diff
@@ -44,15 +38,16 @@ const SnapCardSpacer = styled.div`
 `;
 
 const SnapCardFooter = styled.p`
-    color: var(--text-muted);
+    color: var(--text-main);
     padding: 0;
     margin: 0;
 `;
 
 const SnapList: React.FunctionComponent<SnapListProps> = ({ snaps }) => {
-    function formatAts(ats: String[]): String {
-        return ats.map((s) => `@${s}`).join(" ");
-    }
+    // The regex matches something of the form [full name]
+    const fullNameRegex = /\[[A-Za-z0-9 _]*\]/g;
```

Comment:
Creating a regex to recognise names is _definitely_ a bit of a code smell (at least in part because [you'll never really capture all names](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)) and it is not okay to tell someone that their name is invalid.

We should be able to accomplish this without the need for the a name regex though.